test/compilable/test18199.d

```diff
@@ -0,0 +1,62 @@
+// REQUIRED_ARGS:
+// PERMUTE_ARGS:
+// https://issues.dlang.org/show_bug.cgi?id=18199
+
+//
+// initializer is a struct initializer
+//
+
+// original error report
+struct Bla {
```

Comment:
Although I don't think it should prevent this PR from being pulled, I'd prefer if the code followed the general D coding style, that is curly braces on their own line.

Reply:
done, thank you for raising